Source/hydro/Castro_ctu_rad.cpp

```diff
@@ -122,42 +121,43 @@ Castro::ctu_rad_consup(const Box& bx,
 #endif
     }
 
+
     // we now want to compute the change in the kinetic energy -- we
-    // base this off of uout, since that has the source terms in it.
-    // But note that this does not have the fluxes (since we are
-    // using update)
+    // base this off of S_new, since that already is updated with hydro
 
     // note, we need to include the Z component here too (even
     // for 1- and 2-d), since rotation might be in play
 
-    Real urho_new = uout(i,j,k,URHO) + dt * update(i,j,k,URHO);
+    Real urho_new = update(i,j,k,URHO);
```

Comment:
Also please change the name of `update` here

Reply:
updated